ihatemoney/web.py

```diff
@@ -269,6 +269,11 @@ def home():
         session=session,
     )
 
+@main.route("/download-mobile-app", methods=["GET"])
+def downloadMobileApp():
+    if request.method == "GET":
```

Comment:
Hi, I believe you don't need to have the check for the "GET" method here AND on the route decorator. You could remove this from inside the method :-)

Reply:
Yes you are right! I don't need to check for "GET".